src/main/java/com/cozyhome/onlineshop/userservice/model/Address.java

```diff
@@ -0,0 +1,54 @@
+package com.cozyhome.onlineshop.userservice.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Document(collection = "Address")
+public class Address {
+
+    @Id
+    private String id;
+    private String city;
+    private String street;
+    private String house;
+    private Integer apartment;
+    private Integer entrance;
+    private Integer floor;
+    private boolean withLift;
+    private LocalDateTime createdAt;
+    private LocalDateTime modifiedAt;
```

Comment:
these are not a custom methods, these are overridden methods. these methods are needed so that java knows how to compare this object. Maybe I'm missing something, we need to discuss it in more detail.

Reply:
done